scripts/build_developer.sh

```diff
@@ -0,0 +1,30 @@
+#!/bin/bash
```

Comment:
This is fine with me, as long as we keep it this simple and we continue to keep automated processes using the configs in `infra/ansible`.